### ISSUE TITLE
Build failing with if process package version is < 1.2.1

### ISSUE THREAD

**Issue by tarleb** [jgm/pandoc#2916]
The *process* package prior to 1.2.1 uses `createProcess_` only internally and doesn't export its `Internals` subpackage.  Building with *process* 1.1.0.2 failed. Reinstalling a newer *process* resolved this issue.  Should the cabal file be changed accordingly?

**Comment by jgm** [comment]
I think this is fixed now.

+++ Albert Krewinkel [May 12 16 06:49 ]:

**Comment by jgm** [comment]
OK, looking at the cabal file for 1.1.0.2, I see
```
    if impl(ghc)
        exposed-modules:
          System.Process.Internals
```
So it looks as if it does indeed export System.Process.Internals for ghc.  Are you using a different compiler?  Maybe it's easiest just to depend on a more recent version of process.  Then we could also avoid closing the dummy pipes.  Would depending on a more recent process mess things up for anyone?

**Comment by jgm** [comment]
Fixed by 07a4320ba97cdd219e5cbb18f21dbbda00bc5543 requiring process 1.2.1.

**Comment by tarleb** [comment]
Strange.  The error I got was

```
pandoc.hs:60:35:
    Module `System.Process.Internals' does not export `createProcess_'
```

`cabal list` gave me

```
* process
    Synopsis: Process libraries
    Default available version: 1.4.2.0
    Installed versions: 1.1.0.2
    License:  BSD3
```

Either way, bumping the lower bound fixed it. Thanks!
